core/common/src/main/scala/org/typelevel/otel4s/Attribute.scala

```diff
@@ -86,6 +115,46 @@ object Attribute {
   def apply[A: AttributeKey.KeySelect](name: String, value: A): Attribute[A] =
     Impl(AttributeKey.KeySelect[A].make(name), value)
 
+  /** Creates an attribute with the given name and value.
+    *
+    * @example
+    *   {{{
+    * case class UserId(id: Int)
+    * implicit val userIdKeySelect: Attribute.Focus[UserId, Long] = _.id.toLong
+    * val userIdKey = AttributeKey[Long]("user.id")
+    * val attribute: Attribute[Long] = Attribute.from(userIdKey, UserId(1))
+    *   }}}
+    *
+    * @param key
+    *   the key of an attribute
+    *
+    * @param value
+    *   the value of an attribute
+    */
+  def from[A, Key](key: AttributeKey[Key], value: A)(implicit focus: Focus[A, Key]): Attribute[Key] =
```

Comment:
Are there other alternatives? `from`, `makeFocus`, `focus`?

Reply:
`from` seems good

Reply:
The function is now called `from` , but the conversion trait is still called `Focus`.
Is this intended ?

Reply:
I renamed it to `From` for consistency.